### PR TITLE
Add Privacy & TOS spannables, fix loginActivity for smallscreens

### PR DESCRIPTION
**Pull request by KaenDagger**
Changes: 
- Add Spannables for Privacy & TOS
- Fix overlap in LoginActivity & CompleteProfileActvity

Screenshots for the change:

image

image

### CODE REVIEW

File: app/src/main/java/com/codingblocks/cbonlineapp/auth/onboarding/SignInFragment.kt [hunk 1]
```diff
@@ -19,7 +19,7 @@ import kotlinx.android.synthetic.main.fragment_sign_in.*
 
 class SignInFragment : Fragment() {
 
-    lateinit var type: String
+    var type: String = ""
```
Comment (KaenDagger) [inline app/src/main/java/com/codingblocks/cbonlineapp/auth/onboarding/SignInFragment.kt]: App was crashing due to uninitialized value for this. Also default value was empty string only. So replaced with this

File: app/src/main/java/com/codingblocks/cbonlineapp/auth/onboarding/CompleteProfileActivity.kt [hunk 2]
```diff
@@ -17,4 +21,45 @@ class CompleteProfileActivity : AppCompatActivity() {
             finish()
         }
     }
+
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/auth/onboarding/CompleteProfileActivity.kt]: this method already exist in admin inbox fragment use that

**Comment by aggarwalpulkit596** [comment]
add some more margin for dashboard button

**Comment by aggarwalpulkit596** [comment]
Screenshot 2020-01-08 at 5 41 42 PM
Align With Design

**Comment by KaenDagger** [comment]
Aligned

image

**Comment by KaenDagger** [comment]
Aligned


image

**Pull request merged**